### PR TITLE
product_format_reviewing_note

### PR DESCRIPTION
**Pull request by jposada202020**
Changing Product info format and verifying note on board

### CODE REVIEW

**Comment by jposada202020** [comment]
@kattni as discussed, please verify, I will go through the libraries little by little, 
I will make a PR if the library has other changes in addition to this.

Thanks

File: adafruit_ds3231.py [hunk 1]
```diff
@@ -17,21 +17,26 @@
...
-* Adafruit `DS3231 Precision RTC FeatherWing <https://www.adafruit.com/products/3028>`_
-  (Product ID: 3028)
+* `Adafruit DS3231 Precision RTC FeatherWing
+  <https://www.adafruit.com/products/3028>`_ (Product ID: 3028)
+
+* `Adafruit DS3231 RTC breakout
+  <https://www.adafruit.com/products/3013>`_ (Product ID: 3013)
+
+* `Adafruit ChronoDot - Ultra-precise Real Time Clock - v2.1
+  <https://www.adafruit.com/products/255>`_ (Product ID: 3013)
```
Comment (kattni) [inline adafruit_ds3231.py]: This should be updated, yes?
```suggestion
  <https://www.adafruit.com/products/255>`_ (Product ID: 255)
```

**Review by kattni — CHANGES_REQUESTED** [review]
Looks good! One  suggested edit.

File: adafruit_ds3231.py (hunk 1 above)
Comment (jposada202020) [reply adafruit_ds3231.py]: Thanks :)

**Review by kattni — APPROVED** [review]

File: adafruit_ds3231.py (hunk 1 above)
Comment (jposada202020) [reply adafruit_ds3231.py]: Changes in https://github.com/adafruit/Adafruit_CircuitPython_DS3231/pull/35/commits/03c7efa48c8e39533b2305e0fd68f64649cc1871

**Pull request merged**